File: week-03/day-3/11.py
# Create a function that prints a diamond like this:
#       *
#      ***
#     *****
#    *******
#   *********
#  ***********
#   *********
#    *******
#     *****
#      ***
#       *
#
# It should take a number as parameter that describes how many lines the diamond has
j = 50
k = j
l = j
x=0
for i in range(j):
    print((j)*' '+'*'*(i*2+1))
    j-=1
    x+=2
x=k*2-1
for i in range(k-2):
    print((i+2)*' '+(x-2)*'*')
    k-=1
    x-=2
if l > 0:
    print(l*' '+'*')

File: winter holiday/1.1.js
'use strict';

// Create a function that takes a number and returns it as string in english
// like 0 -> "zero", it should work with the first 5 numbers, after it should
// return "many"

var number = 1;
function e(number){
    switch (number){
      case 1:
        console.log('one');
        break;
      case 2:
        console.log('two');
        break;
      case 3:
        console.log('three');
        break;
      case 4:
        console.log('four');
        break;
      case 5:
        console.log('five');
        break;
      default:
        console.log('javascript is shit');
    }};
e(number);
e(number+1);
e(number+2);
e(number+3);
e(number+4);
e();
